exporter/awsemfexporter/util.go

```diff
@@ -172,3 +173,14 @@ func attrMaptoStringMap(attrMap pcommon.Map) map[string]string {
 	})
 	return strMap
 }
+
+func processAttributes(entityMap map[string]string, targetMap map[string]*string, mutableResourceAttributes pcommon.Map) {
```

Comment:
Please add a comment here to describe why we're doing this, particularly the part where we're removing the entity attribute after processing it.

Reply:
Addressed in latest commit